tests/test_pandas.py

```diff
@@ -129,11 +130,11 @@ def test_isna() -> None:
     idx2 = pd.Index([1, 2])
     check(assert_type(pd.notna(idx2), np.ndarray), np.ndarray)
 
-    check(assert_type(pd.isna(pd.NA), bool), bool)
-    check(assert_type(pd.notna(pd.NA), bool), bool)
+    check(assert_type(pd.isna(pd.NA), Literal[True]), bool)
+    check(assert_type(pd.notna(pd.NA), Literal[False]), bool)
```

Comment:
For completeness here, you should be able to do 
```python
assert check(assert_type(pd.isna(pd.NA), Literal[True]), bool)
assert not check(assert_type(pd.notna(pd.NA), Literal[False]), bool)
```
and likewise in the next 2 lines

This will make sure we are returning the right result.